src/commands/autocomplete/create.ts

```diff
@@ -39,6 +40,7 @@ export default class Create extends AutocompleteBase {
     await fs.writeFile(this.bashCompletionFunctionPath, this.bashCompletionFunction)
     await fs.writeFile(this.zshSetupScriptPath, this.zshSetupScript)
     await fs.writeFile(this.zshCompletionFunctionPath, this.zshCompletionFunction)
+    await fs.writeFile(this.fishCompletionFunctionPath, this.fishCompletionFunction)
```

Comment:
Since this is writing to a file we don't own, we should only run it if fish shell is passed as an arg or detected and also warn the user.

Reply:
Is the same true of the `zsh` and `bash` completion files? I used them as a template for this PR and didn't see anything similar for how to detect that it was passed in as an arg. Could you give me some guidance on how to detect that it was passed in as an arg? Thanks!

Reply:
We own the zsh and bash completion directories. `this.config.shell` will get you the user's shell. We can skip detecting the shell as an arg for now since `create` is ran for the user.

Reply:
Cool, I've pushed 068bd30 now which I believe addresses this. It wraps the writing of the fish file in a conditional that checks that the shell was `fish` and it also emits a debug log that fish shell was detected and where we're writing the autocomplete to.

Reply:
Cool, I've pushed 068bd30 now which I believe addresses this. It wraps the writing of the fish file in a conditional that checks that the shell was `fish` and it also emits a debug log that fish shell was detected and where we're writing the autocomplete to.